### ISSUE TITLE
Questions on FS2 evaluation and MB-Melgan finetune

### ISSUE THREAD

**Issue by PedroDKE** [TensorSpeech/TensorFlowTTS#535]
For fastspeech2:
Is there a particular reason why the split is only done in train/val and not in train/val/test ?

Mb-Melgan:
So i'd like to finetune the MB-melgan model trained on the librispeech dataset (the only model with 24khz sample rate) for my own speakers. When i used the provided config the generated audio actually gets worse instead of better after 150k iterations. I assume it is overfitting since the starting model has already an okay performance. So i wanted to ask which parameters in the config file i should change ? I'm assuming I should/could lower the learning rate, but should i start training the discriminator already from step 0 too?

**Comment by dathudeptrai** [comment]
> Is there a particular reason why the split is only done in train/val and not in train/val/test ?

There are no particular reason :D, you can do it if you want.

> So i'd like to finetune the MB-melgan model trained on the librispeech dataset (the only model with 24khz sample rate) for my own speakers. When i used the provided config the generated audio actually gets worse instead of better after 150k iterations. I assume it is overfitting since the starting model has already an okay performance. So i wanted to ask which parameters in the config file i should change ? I'm assuming I should/could lower the learning rate, but should i start training the discriminator already from step 0 too?

Yes, you should training discriminator from step 0. About learning rate for generator, let decrease it :D.
